apps/resources/dotty-repl.json

```
@@ -4,7 +4,7 @@
     "central"
   ],
   "dependencies": [
-    "ch.epfl.lamp:dotty-compiler_0.26:latest.stable"
+    "org.scala-lang:scala3-compiler_3.0.0-M3:latest.stable"
```

I believe we should use `latest.release` rather than `latest.stable` here. The latter rejects milestones / RCs / etc. so it would reject `3.0.0-M3` here.

```suggestion
    "org.scala-lang:scala3-compiler_3.0.0-M3:latest.release"
```